nservicebus/transports/guidance.md

```diff
@@ -0,0 +1,118 @@
+---
+title: SQL Server Transport usage guidance
+summary: Usage guidance for SQL Server transport
+tags:
+- SQL Server
+---
+
+The SQL Server transport implements a message queueing mechanism on top of Microsoft SQL Server. It provides support for sending messages over [SQL Server](http://www.microsoft.com/en-au/server-cloud/products/sql-server/) tables. It does **not** make any use of ServiceBroker, a messaging technology built into the SQL Server.
+
+
+## How it works
+
+The SQL Server transport is a hybrid queueing system which is neither store-and-forward (like MSMQ) nor a broker (like RabbitMQ). It treats the SQL Server as a storage infrastructure for the queues while the queue implementation logic is executed inside the client process.
+
+
+### Advantages of SQL Server Transport
+
+ * No additional licensing and training costs, as majority of Microsoft stack organizations already have SQL Server installed and have knowledge required to run it
+ * Great general-purpose tooling (SSMS)
+ * Throughput on par with MSMQ
+ * Scaling out of processing is simple because of support for competing consumers at the queue level
+
+## Deployment considerations
+
+### Introduction
+Any NServiceBus endpoint operates and manages different types of data, those are:
+ * Business data - data used for implementing business capabilities
+ * Persistence data - infrastructure level data including: saga data, timeout manager state and message driven pub/sub information
+ * Transport data - messaging infrastructure state
+
+SQL Server Transport manages transport data and puts no constraints on the type and configuration of storage technology used for persistence and business data. It can work with any of available persisters i.e. NHibernate and RavenDB, as well as any storage mechanisms used inside message handlers.
+
+Building any system based on NServiceBus requires answering questions on how and where business, persistence and transport data will be stored. The purpose of this section is to define what needs to be considered from SQL Server Transport's perspective.
+ 
+NOTE: No matter what deployment options are chosen, there is one general rule that should always apply: **All transport data (input, audit and error queues) should reside on a single SQL Server instance**. Multi-instance deployments are technically possible but have been deprecated as of SQL Server Transport version 3. Please see: ###Section### for migration guidance.  
```

Comment:
After giving it some thought, highlighting this information should be enough. We're good.